### DISCUSSION TITLE
Mountain Duck and Verizon Cloud?

### DISCUSSION THREAD

**Discussion by JPUnix** [iterate-ch/cyberduck#16287] [Q&A] [answered]
Can I use Mountain Duck to attach my Verizon Cloud virtual drive so it can appear on my PC as a network drive?

TIA!

**Comment by dkocher** [comment]
Can you point us to the service description. It would require to provide a public API that Mountain Duck supports such as WebDAV or S3.

**Reply by JPUnix** [reply]
It's Verizon Cloud, a subscription service offered by Verizon. Unfortunately I don't have any more details. Hence my question.

**Reply by dkocher** [reply, marked as answer]
We previously had support for Verizon Cloud Storage (object storage service compatible with S3 API) but that service no longer seems to be available. [Verizon Cloud](https://www.verizon.com/solutions-and-services/add-ons/protection-and-security/verizon-cloud/) seems to refer to a mobile backup service we are unable to support.

**Reply by JPUnix** [reply]
That's what I suspected also. Thanks for the confirmation!

**Comment by x86txt** [comment]
FWIW, they switched from using direct S3 to a whitelabel provider called Synchronoss (who uses AWS S3), but you login with Verizon's SSO, so even though I can find the S3 endpoint, I don't see a way to login.
